### PR TITLE
Feature/rb 87 system send life update

### PR DESCRIPTION
**Pull request by romainpanno**
* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Norm of the code has been respected

* **In what state is your pull request?**
- [x] Ready to merge / Waiting a reviwer to see my work.
- [ ] Work In Progress (WIP) / My work is not finish but i want daily reviews. (Draft)
- [ ] CI Review / I want to see what the CI thinks of my work.
- [x] Need feedback / Just to have feedback on what i produced. (May not be merged)

* **What kind of change does this PR introduce?** (You can choose multiple)
- [ ] Bug fix
- [x] Feature request
- [ ] New / Updated documentation
- [ ] Testing CI ( Make the pull request in draft mode)

* **What is the current behavior?** (link an issue based on the kind of change this pr introduce)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

**Review by kw4ntiq — APPROVED** [review]

File: src/Client/Systems/Events/EventsSystems.cpp [hunk 1]
```diff
@@ -6,7 +6,9 @@
 */
 
 #include "EventsSystems.hpp"
+#include <iostream>
```
Comment (Saverio976) [inline src/Client/Systems/Events/EventsSystems.cpp]: iostream ?

File: src/Client/Systems/Events/EventsSystems.cpp [hunk 2]
```diff
@@ -62,58 +64,61 @@ namespace Systems {
...
+        Registry::getInstance().getComponents<Types::CollisionRect>().insertBack(collisionRect1);
+        Registry::getInstance().getComponents<Types::Rect>().insertBack(spriteRect);
+        Registry::getInstance().getComponents<Types::CollisionRect>().insertBack(collisionRect2);
+        Registry::getInstance().getComponents<Types::Missiles>().insertBack(missileType);
+        Registry::getInstance().getComponents<Types::PlayerAllies>().insertBack(playerAlliesComp);
+        Registry::getInstance().getComponents<Types::Velocity>().insertBack(velocity);
+        Registry::getInstance().getComponents<health_s>().insertBack(healthComp);
+        Registry::getInstance().getComponents<Types::Damage>().insertBack(damageComp);
+        Registry::getInstance().getComponents<Types::Dead>().insertBack(deadComp);
+        std::cout << "missile id: " << entityId << std::endl;
```
Comment (Saverio976) [inline src/Client/Systems/Events/EventsSystems.cpp]: cout

File: src/ECS/Registry.cpp [hunk 3]
```diff
@@ -6,6 +6,7 @@
 */
 
 #include "Registry.hpp"
+#include <iostream>
```
Comment (Saverio976) [inline src/ECS/Registry.cpp]: iostream

File: src/ECS/Registry.cpp [hunk 4]
```diff
@@ -35,6 +50,42 @@ void Registry::removeEntity(std::size_t id)
...
+    }
+    for (auto &layer : _backLayers) {
+        removeEntityFromLayer(id, layer);
+    }
+    removeEntityFromLayer(id, _defaultLayer);
+    ;
+    for (auto &layer : _frontLayers) {
+        removeEntityFromLayer(id, layer);
+    }
+    std::cout << "after removeEntity" << std::endl;
```
Comment (Saverio976) [inline src/ECS/Registry.cpp]: cout

File: src/ECS/Systems/Systems.cpp [hunk 5]
```diff
@@ -8,12 +8,18 @@
 #include "Systems.hpp"
 #include <cstddef>
 #include <fstream>
+#include <iostream>
```
Comment (Saverio976) [inline src/ECS/Systems/Systems.cpp]: iostream

**Review by Saverio976 — APPROVED** [review]

**Review by guillaumeAbel — APPROVED** [review]

**Pull request merged**